spec/factories/recruitment_cycles.rb

```diff
@@ -8,6 +8,10 @@
       year { (Settings.current_recruitment_cycle_year.to_i - 1).to_s }
     end
 
+    trait :year_before_last do
+      year { (Settings.current_recruitment_cycle_year.to_i - 2).to_s }
+    end
+
```

Comment:
seems like the only time we use a RC two years prior is for the previous allocation cycle. perhaps `previous_allocation_cycle` is a better-suited name for this trait? 

```suggestion
   trait :previous_allocation_cycle do
      year { (Settings.allocation_cycle_year - 1).to_s }
    end

```